ui/src/sandbox.rs

```diff
@@ -280,6 +281,27 @@ fn set_execution_environment(
     cmd.apply_backtrace(&req);
 }
 
+fn parse_magic_comments(code: &str) -> Vec<(&'static str, String)> {
+    lazy_static::lazy_static! {
+        pub static ref MIRIFLAGS: Regex = Regex::new(r#"(///|//!|//)\s*MIRIFLAGS\s*=\s*(.*)"#).unwrap();
+    }
+
+    fn match_of(line: &str, pat: &Regex) -> Option<String> {
+        pat.captures(line.trim())
+            .and_then(|caps| caps.get(2))
+            .map(|mat| mat.as_str().trim_matches(&['\'', '"'][..]).to_string())
```

Comment:
Hoe does this handle trailing whitespace?

Reply:
Trailing whitespace outside quotes should be trimmed off, trailing whitespace inside quotes is currently retained I think. Is that a problem?

Reply:
No that's fine, I missed the `line.trim()` input beforehand 👍